src/sec_adapter_processor.c

```diff
@@ -340,11 +340,8 @@ Sec_Result SecProcessor_GetInfo(Sec_ProcessorHandle* processorHandle, Sec_Proces
     if (secProcInfo == NULL)
         return SEC_RESULT_INVALID_PARAMETERS;
 
-    sa_version version;
-    sa_status status = sa_get_version(&version);
-    CHECK_STATUS(status)
+    memcpy(secProcInfo->version, SEC_API_VERSION, sizeof(SEC_API_VERSION));
```

Comment:
Does this give us enough information in logs, telemetry, and analytics to determine that the build is using a secapi2-adapter + secapi3 implementation instead of a stock secapi2 based build? Why does SOC provisioning need to know the secapi version information?